### DISCUSSION TITLE
XRD texture and Complete pole figure

### DISCUSSION THREAD

**Discussion by lyg1232** [mtex-toolbox/mtex#1468] [Ask Anything]
Hello! The following incomplete pole figure is obtained by calculation, but I don't know how to get the calculated complete pole figure by measuring the pole figure, as well as the corresponding inverse pole figure, odf figure. Thank you !
![微信图片_20220627201250](https://user-images.githubusercontent.com/81472625/175938870-75a7d8a1-6a37-4f0f-bcdc-71fe1b6df70c.jpg)

**Comment by kilir** [comment]
Hi,
could you try to be a little more detailed? You measured a set of incomplete pole figures and would like to obtain an odf?
See here:  https://mtex-toolbox.github.io/PoleFigure2ODF.html
Cheers,
Rüdiger

**Reply by lyg1232** [reply]
Hi, The four incomplete pole figures measured by XRD, I want to convert the incomplete pole figures into complete pole figures,Can you tell me?

**Reply by kilir** [reply]
Yes, have a look here:
 https://mtex-toolbox.github.io/PoleFigure2ODF.html
Cheers,
Rüdiger
